page-officers.hbs

```diff
@@ -332,6 +332,15 @@ into the {body} of the default.hbs template --}}
                     <a target="_blank" class="icon-github" href="https://github.com/DarKnight0102/"></a>
                   </div>
                 </div>
+                <div class="ieee-officer-card">
+                  <img src="{{asset 'officers/Website/AndrewZhang142x142.jpg'}}" alt="profilepic"/>
```

Comment:
Sorry! Looks like I was unclear, the file path should match that of the other files within the folder.